### PR TITLE
build: run release-please app on Cloud Run

### PR DESCRIPTION
**Pull request by chingor13**
Towards #4157

### CODE REVIEW

File: packages/release-please/src/server.ts [hunk 1]
```diff
@@ -12,22 +12,13 @@
...
 
 const bootstrap = new GCFBootstrapper({
-  taskTargetEnvironment: 'functions',
+  taskTargetEnvironment: 'run',
 });
-
-// We only need to deploy gcf-utils in the frontend server because
-// only thing it does is to enqueue a task for the release-please
-// backend. Thus we deploy a dummy empty bot code.
-
```
Comment (tmatsuo) [inline packages/release-please/src/server.ts:26]: So are you planning to use this file for both frontend and backend?

**Review by tmatsuo — APPROVED** [review]

**Review by tmatsuo — CHANGES_REQUESTED** [review]
I think you also need to modify the cloudbuild.yaml.

File: packages/release-please/src/server.ts (hunk 1 above)
Comment (chingor13) [reply packages/release-please/src/server.ts:26]: Yes, we are currently running a frontend, but we will have a single service that handles both.

**Comment by chingor13** [comment]
> I think you also need to modify the cloudbuild.yaml.

We are already deploying it to GCF and Cloud Run. If this works as expected, I'll submit a follow up PR that stops building/deploying to GCF

**Comment by tmatsuo** [comment]
I still recommend we have a Cloud Run frontend service **and** another Cloud Run backend service because those services have different scaling requirements.

File: packages/release-please/src/server.ts (hunk 1 above)
Comment (chingor13) [reply packages/release-please/src/server.ts:26]: Now we are running separate frontend/backend.

**Pull request merged**